docs/tutorial/security.md

```diff
@@ -278,7 +278,8 @@ session
       callback(true)
     }
 
-    if (!url.startsWith('https://my-website.com')) {
+    // Verify URL
+    if (!url.startsWith('https://my-website.com/')) {
```

Comment:
how does that extra ```/``` benefit?

Reply:
As I said in #16774, the current example (if followed by the reader) can be bypassed with an URL which starts with the required string, but may be a subdomain (or part of the domain) like `https://my-website.com.attacker.com`, which passes the check. 
Also, this follows the example provided in point 11 (https://github.com/electron/electron/blob/master/docs/tutorial/security.md#how-9). Another option would be to parse the URL and strictly check the origin like in point 12 (https://github.com/electron/electron/blob/master/docs/tutorial/security.md#how-10).